testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc="nodejs bash"
+makedepends="linux-headers"
+install=""
+subpackages="$pkgname-doc"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
+	zerotier-one.initd
+	"
+
+builddir="$srcdir"/ZeroTierOne-$pkgver
+build() {
+	cd "$builddir"
+
+	# build base binaries
+	make DESTDIR="$builddir" one || return 1
+
+	# link all the binaries
+	make DESTDIR="$builddir" install || return 1
+}
+
+package() {
+	cd "$builddir"
+
+	# setup init script
+	install -m755 -D "$srcdir"/$pkgname.initd \
+		"$pkgdir"/etc/init.d/$pkgname || return 1
+
+	# post-install message
+	cat <<EOF
+To start zerotier-one:
+---------------------
+sudo rc-update add zerotier-one
+sudo rc-service zerotier-one start
+
+then...
+zerotier-cli join <network>
+
+EOF
+}
+
+doc() {
```

Comment:
No need to specify a doc function abuild will scan and take care of docs itself (if you added it to $subpackages ofc).